tokio-test/tests/io.rs

```diff
@@ -1,6 +1,9 @@
 #![warn(rust_2018_idioms)]
 
-use std::io;
+use std::{
+    io,
+    time::{Duration, Instant},
+};
```

Comment:
We don't use that import style in Tokio.
```suggestion
use std::io;
use std::time::{Duration, Instant};
```